troubleshooting-guide.md

```diff
@@ -52,6 +54,77 @@ Welcome to the troubleshooting guide for Azure Functions Dapr Extension. This gu
 
 **Preventive Measures:** If you don't need to debug the extension, you should always use [published nuget package](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Dapr)
 
+## Dapr sidecar is not present
+
+**Symptoms:** If you are utilizing Dapr bindings and triggers in Azure Functions, you might encounter an error message like the following:
+
+```plaintext
+Dapr sidecar is not present. Please see (https://aka.ms/azure-functions-dapr-sidecar-missing) for more information.
+```
+
+**Possible Causes:** This error typically occurs when Dapr is not properly enabled in your environment.
+
+**Resolution:**
+
+***Enable Dapr in your application:***
+
+- If your Azure Function is deployed in Azure Container Apps (ACA), refer to the documentation [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-dapr?tabs=in-process%2Cpreview-bundle-v4x%2Cbicep1&pivots=programming-language-python#dapr-enablement) for instructions on enabling Dapr.
+- If your Azure Function is deployed in Kubernetes, ensure that your [deployment](https://github.com/ASHIQUEMD/azure-functions-dapr-extension/blob/master/deploy/kubernetes/kubernetes-deployment.md#sample-kubernetes-deployment) has the following annotations in its YAML configuration:
+    ```YAML
+    annotations:
+        dapr.io/enabled: "true"
+        dapr.io/app-id: "functionapp"
+        # Only define port of Dapr triggers are included
+        dapr.io/app-port: "3001"
+    ```
+- If you are running your Azure Function locally, make sure you are [running the function app with Dapr](https://github.com/ASHIQUEMD/azure-functions-dapr-extension/tree/master/samples/python-v2-azurefunction#step-2---run-function-app-with-dapr). Execute the following command:
+    ```bash
+    dapr run --app-id functionapp --app-port 3001  --components-path ..\components\ -- func host start 
+    ```
+
+**Preventive Measures:** To prevent this error, always ensure that Dapr is properly set up and enabled in your environment before using Dapr bindings and triggers with Azure Functions.
+
+## Dapr app config error
+
+**Symptoms:** If you encounter the following error message:
+```plaintext
+The Dapr sidecar is configured to listen on port {portInt}, but the app server is running on port {appPort}. This may cause unexpected behavior. For more information, visit [this link](https://aka.ms/azfunc-dapr-app-config-error).
+```
+
+**Possible Causes:** This error can occur when using Dapr Triggers. The Azure Function Dapr Extension starts an HTTP server on port 3001 when there is Dapr trigger in your Azure Function, you can configure this port using the DAPR_APP_PORT environment variable. If you provide an incorrect app_port value when running the Function app, it can lead to this problem.
+
+**Resolution:**
```

Comment:
This part should contain the information on customizability with `DAPR_APP_PORT`.

Reply:
Feel free to modify this content, I will incorporate your commit.